pkg/bucket/bucket.go

```diff
@@ -239,25 +243,30 @@ func RunList(cmd *cobra.Command, args []string) {
 	fmt.Printf("\n")
 }
 
-func prepareQuotaConfig(maxSize string, maxObjects string) (nb.QuotaConfig, bool) {
+func prepareQuotaConfig(bucketName string, maxSize string, maxObjects string) (nb.QuotaConfig, bool, error) {
 	var bucketMaxSize, bucketMaxObjects int64
-	updateQuota := false
+	quota := nb.QuotaConfig{}
+
+	// validate quota config for bucket
+	err := util.ValidateQuotaConfig(bucketName, maxSize, maxObjects)
+	if err != nil {
+		return quota, false, err
+	}
 
-	if len(maxSize) > 0 {
-		bucketMaxSize, _ = strconv.ParseInt(maxSize, 10, 64)
-		updateQuota = true
+	if maxSize != "" {
+		quantity, _ := resource.ParseQuantity(maxSize)
+		bucketMaxSize = quantity.Value()
 	}
-	if len(maxObjects) > 0 {
+	if maxObjects != "" {
 		bucketMaxObjects, _ = strconv.ParseInt(maxObjects, 10, 64)
-		updateQuota = true
 	}
-	quota := nb.QuotaConfig{}
+
 	if bucketMaxSize > 0 {
 		f, u := nb.GetBytesAndUnits(bucketMaxSize, 2)
 		quota.Size = &nb.SizeQuotaConfig{Value: f, Unit: u}
 	}
 	if bucketMaxObjects > 0 {
 		quota.Quantity = &nb.QuantityQuotaConfig{Value: int(bucketMaxObjects)}
 	}
-	return quota, updateQuota
+	return quota, true, err
```

Comment:
2 return values quota, update_quota are enough. Is n't it.

Reply:
yes we can return `quota, update_quota=true, err=nil` (as there is no error) here. Thanks